vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "redirects": [
+    {
+      "source": "/",
+      "destination": "scaling/tvl",
+      "permanent": true
+    }
```

Comment:
What about other redirects?

Reply:
I think all other redirects were configured for legacy reasons.

Reply:
Not the api one